### ISSUE TITLE
Add support for Dutch AQI

### ISSUE THREAD

**Issue by justb4** [hrbonz/python-aqi#5]
Within the Netherlands the [National Institute for Public Health and the Environment - RIVM](http://rivm.nl/en/)   has developed a local AQI, called "LKI", based on NO2, O3 and PM10 (and PM2.5?). See:
http://www.rivm.nl/Onderwerpen/S/Smog/Waarschuwingsgrenzen_en_luchtkwaliteitsindex

![image](https://user-images.githubusercontent.com/582630/32793581-a7353840-c966-11e7-8dce-d0003d96316d.png)

It would be useful to extend the current `python-aqi` algorithms with `DutchRIVM`. I need to figure out the exact algorithms first though. A use would be in the [Smart Emission Project](http://data.smartemission.nl) and [the issue there](https://github.com/Geonovum/smartemission/issues/67).

I hope to contribute via a PR, but maybe others can help me finding the exact calculations for the algorithm.

**Comment by hrbonz** [comment]
If you can provide me with the formula concentration -> RIVM, I can give a hand.

**Comment by justb4** [comment]
Thanks! Yes, found some [more details, even in English](https://www.luchtmeetnet.nl/uitleg#), though it doesn't expand the formula in English, but only in the Dutch version:

![image](https://user-images.githubusercontent.com/582630/32884475-c8b23f4a-caba-11e7-90dc-f2a7483d42c0.png)

The [full RIVM report is here (106 pages)](http://www.rivm.nl/Documenten_en_publicaties/Wetenschappelijk/Rapporten/2015/mei/Luchtkwaliteitsindex_Aanbevelingen_voor_de_samenstelling_en_duiding) though in Dutch as well but the [PDF ](http://www.rivm.nl/dsresource?objectid=2f77619a-00ce-4d0c-8c35-341603decf97&type=org&disposition=inline) can't be opened at the moment....

In Dutch:
"*Zowel de gemeten als berekende luchtkwaliteit wordt ingedeeld in vijf klassen van goed tot zeer*
*slecht. Per stof wordt een concentratie omgezet tot een getal van 1 (weinig luchtverontreiniging)* *tot 11 (veel luchtverontreiniging). De stof met het hoogste indexgetal bepaalt de totale*
*luchtkwaliteitsindex. Als alle stoffen hetzelfde indexgetal hebben dan wordt de totale index één punt hoger."*

Trying to convert this to English:

* there are 5 classes in the index (though expanded to 11 here)
* 4 components are taken: NO2, O3, PM10 and PM2.5
* per component a concentration in ug/m3 is converted to a number 1-11 (see table above)
* these numbers are ranked
* the component with the highest rank determines the index number
* if __all__ components have the same index number the resulting index (AQI) will be one higher

Only thing I see that there are three ranges: 1-5, 1-11 and even 1-17 in the legend below, though this is refinement on the 1-11 index, so I think it should range 1-11.

![image](https://user-images.githubusercontent.com/582630/32885367-9653fa40-cabd-11e7-8048-80e3d1240766.png)

So basically we have all the info I think. Now we can see about the next step.

**Comment by hrbonz** [comment]
I'm back working on the project so I'll take a look at this this weekend.
